### ISSUE TITLE
LISA显存占用

### ISSUE THREAD

**Issue by DYF-AI** [modelscope/ms-swift#4522]
**Describe the bug**
使用LISA对Qwen2.5VL-3B模型进行微调，--lisa_activated_layers 1 \--lisa_step_interval 20 \，显存占用达到32GB
{'loss': 2.35480309, 'token_acc': 0.61641221, 'grad_norm': 69.5, 'learning_rate': 1.429e-05, 'memory(GiB)': 31.33, 'train_speed(iter/s)': 0.000522, 'epoch': 0.01, 'global_step/max_steps': '1/125', 'percentage': '0.80%', 'elapsed_time': '31m 54s', 'remaining_time': '2d 17h 55m 50s'}

使用LoRA训练只需要23GB作用，和论文提到和LoRA显存占用差不多，有点矛盾？
